src/main/webapp/js/form.js

```diff
@@ -1,61 +1,43 @@
-//This file handles the formatting for the form on RRP Homepage
-//This is done with a third-party formatting and styling package
-(function ($) {
-    'use strict';
-    /*==================================================================
-        [ Daterangepicker ]*/
-    try {
-        $('.js-datepicker').daterangepicker({
-            "singleDatePicker": true,
-            "showDropdowns": true,
-            "autoUpdateInput": false,
-            locale: {
-                format: 'DD/MM/YYYY'
-            },
-        });
-
-        var myCalendar = $('.js-datepicker');
-        var isClick = 0;
-
-        $(window).on('click',function(){
-            isClick = 0;
-        });
-
-        $(myCalendar).on('apply.daterangepicker',function(ev, picker){
-            isClick = 0;
-            $(this).val(picker.startDate.format('MM/DD/YYYY'));
-
-        });
-
-        $('.js-btn-calendar').on('click',function(e){
-            e.stopPropagation();
-
-            if(isClick === 1) isClick = 0;
-            else if(isClick === 0) isClick = 1;
-
-            if (isClick === 1) {
-                myCalendar.focus();
-            }
-        });
-
-        $(myCalendar).on('click',function(e){
-            e.stopPropagation();
-            isClick = 1;
-        });
-
-        $('.daterangepicker').on('click',function(e){
-            e.stopPropagation();
-        });
-
+/* ==========================================================================
+   SEARCH FORM
+   ========================================================================== */
+// AJAX form POST

```

Comment:
redundant comment

Reply:
done

Reply:
the comment is still here

Reply:
that's weird, I thought I pushed this. thanks for letting me know.